interface/patient_file/summary/demographics.php

```diff
@@ -544,7 +546,13 @@ function setMyPatient() {
 <?php if (isset($_GET['set_pid'])) { ?>
  parent.left_nav.setPatient(<?php echo "'" . addslashes($result['fname']) . " " . addslashes($result['lname']) .
     "'," . addslashes($pid) . ",'" . addslashes($result['pubpid']) .
-    "','', ' " . xls('DOB') . ": " . addslashes(oeFormatShortDate($result['DOB_YMD'])) . " " . xls('Age') . ": " . addslashes(getPatientAgeDisplay($result['DOB_YMD'])) . "'"; ?>);
+    "','', ' " . xls('DOB') . ": " . addslashes(oeFormatShortDate($result['DOB_YMD'])) . " ";
+    $date_of_death = is_patient_deceased($pid)['date_deceased'];
+    if(empty($date_of_death)) {
+        echo xls('Age') . ": " . addslashes(getPatientAgeDisplay($result['DOB_YMD'])) . "'";
+    } else {
+        echo xls('Age at time of death') . ": " . addslashes(oeFormatAge($result['DOB_YMD'], $date_of_death)) . "'";
```

Comment:
Would a display like this be more useful for staff?  Not even sure if age at the time of death is that critical.
**`DoB:12/31/1900 Deceased:12/31/2000 (100 Yrs)`**

Reply:
Sure @mdsupport , that looks better